### PR TITLE
Login functionality - fixes and tests added

### PR DESCRIPTION
**Pull request by srishtihunjan**
- Updated restaurant data being inserted via insertRestaurant.js
 - Unit test for registration added
 - Added check for "username already exists"
 - Improved UI of Registration page
 - Removed commented code and extra console.logs. Though I still kept some of them for debugging. TO-DO - set a debug printing standard for uniformity
 - Updated package.json and readme

### CODE REVIEW

File: routes/insertRestaurant.js [hunk 1]
```diff
@@ -24,37 +24,40 @@ router.get('/', function(req, res, next){
...
-            {
-                name:'Masala Papad',
-                description:'Lassi ke saath aa jaye mazaa'
-            },
-            {
-                name:'Chicken Tikka',
-                description:'Kyun de rhe ho bechare chicken ko sazaa'
-            }]        
+                 name:'Burrito',
+                 description:'Stuffed with meat, cheese, rice, beans and sauce. Then smothered with extra sauce and cheese.'
```
Comment (GITANG1) [inline routes/insertRestaurant.js:39]: we might need a image url in the schema to display images.

File: src/app/register/register.component.scss [hunk 2]
```diff
@@ -1 +1,25 @@
...
+}
+
+  .alert-danger {
+    color: #a94442;
+    background-color: #f2dede;
+    border-color: #ebccd1;
+    margin-bottom: 20px;
+    border: 1px solid transparent;
+    border-radius: 4px;
+}
```
Comment (GITANG1) [inline src/app/register/register.component.scss:25]: EOD required

File: routes/insertRestaurant.js (hunk 1 above)
Comment (srishtihunjan) [reply routes/insertRestaurant.js:39]: do we need to add that now itself? Since we aren't planning on storing and working on images

**Review by GITANG1 — APPROVED** [review]
we can create a issue to include image url in the schema.

current files seems good. 
can be merged.

File: routes/insertRestaurant.js (hunk 1 above)
Comment (nakhe93) [reply routes/insertRestaurant.js:39]: I think as of now, we don't require image url

**Review by nakhe93 — APPROVED** [review]
Looks good

**Review by gdeepchadha — APPROVED** [review]

**Pull request merged**